JS/Lesson26/onur/createNewPost/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>New Post</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+
+<body>
+    <div class="container">
+        <h1>Create a new post</h1>
+        <form id="new-post" method="post" novalidate>
+            <div class="form-container">
+                <label for="new-title">Title :</label>
+                <input type="text" name="title" id="new-title" class="text" required>
+            </div>
+            <div class="form-container">
+                <label for="new-body">Body :</label>
+                <textarea type="text" name="body" id="new-body" rows="7" class="text" required></textarea>
```

Comment:
Try avoid setting size of an input field directly in HTML by using "rows", use CSS instead. No need to change here.